src/frontend/src/pages/WorkPackageForm/CreateWorkPackageForm.tsx

```diff
@@ -5,18 +5,23 @@ import { useCreateSingleWorkPackage } from '../../hooks/work-packages.hooks';
 import { useHistory } from 'react-router-dom';
 import { routes } from '../../utils/routes';
 import { projectWbsPipe } from '../../utils/pipes';
+import CreateWorkPackageCRForm from './CreateWorkPackageCRForm';
 
 const CreateWorkPackageForm: React.FC = () => {
   const query = useQuery();
-  const crId = query.get('crId');
+  const crId = query.get('crId') || undefined;
```

Comment:
instead of doing this this way, see where `CreateWorkPackageForm` is being called in in reference to a URL and make a new line for CreateWorkPackageCRForm with a new URL rather than doing it with URL params